internal/venice-common/src/main/java/com/linkedin/venice/meta/DataReplicationPolicy.java

```diff
@@ -30,6 +30,7 @@ public enum DataReplicationPolicy {
    * Samza job per colo pushes to local real-time topic. Leader SNs replicate data from real-time topic in all colos to
    * local version topic.
    */
+  @Deprecated
   ACTIVE_ACTIVE(3);
```

Comment:
Comment here just for reference.

I couldn't find the code change to the lag measurement for AA stores and `NON_AGGREGATE` DRP. Is that expected?

Reply:
We removed use of DRPs for the lag measurement of AA enabled stores